CONTRIBUTING.md

```diff
@@ -0,0 +1,149 @@
+For example, if I want to work on a `feature` called `login-form`, the branch name will be **feature/login-form**. If it is going to be a `fix` in `navbar`, the branch name will be `fix/navbar`.
+Command example:
+
+```
+git checkout -b feature/login-form
+```
+
+6. **Just do it!**
+
+Perform the tasks you wanted to, can be anything, ranging from fixing simple typo to re-designing the whole page!
+
+7. **Committing your changes**
+
+Now you have made the changes, though they are saved in your system, Git doesn't know what changes you've done. So you have to **commit** your changes. First step is to add the files which you want to add to the staging area, the dot after **add** in the first command tells Git to check for changes in all the files. The second step is about committing your changes. The message part is short description of your commit, like "adds a login form on homepage". Please make sure NOT to have commit messages like "fix issue#34". When we look at the commit history, we should understand what a particular commit is supposed to do based on the commit message.
+
+```
+git add .
+git commit -m "Write message about your commit"
+```
+
+8. **Making sure you have the latest changes from the develop-ember branch**
+
+It may so happen that since the last time you cloned the repo/took a pull from develop, some changes may be merged in the develop-ember branch. So to be on the safer side, we should have those changes as well.
+
+In order to do that, we first checkout to **develop-ember** branch by:
+
+```
+git checkout develop-ember
+```
+
+Once we're in develop-ember, it's time to take a pull:
+
+```
+git pull upstream develop-ember
+```
+
+Now that our **local** develop-ember branch is in sync with **remote** develop-ember branch (of the Real Dev Squad Repository), we should let our branch know about the changes from the develop-ember branch (if any). To do so we first checkout to our branch:
+
+```
+git checkout <branch-name>
+```
+
+Once we're in our branch, we **rebase** our branch on top of the current develop-ember branch (we change the base of our branch, so that it appears as if we have worked from the time the latest changes were merged in the develop-ember branch). To do so:
+
+```
+git rebase develop-ember
+```
+
+You should solve the merge conflicts, if any.
+
+9. **Pushing the code**
+
+Now that we have made our changes plus we have the latest changes made by other contributors, we should push our code from **local** branch to the same branch on our **GitHub fork**. We do so by:
+
+```
+git push origin <branch-name>
+```
+
+For example, if the branch name is `feat/login-form`, we enter `git push origin feat/login-form`
+
+The **origin** refers to your GitHub fork. You can check it by entering `git remote -v`, you should the link to your fork against **origin**.
```

Comment:
Sentence can be better formed here. What do you say?
`you should get the link to your fork`

Reply:
You are right about the sentences will read the documentation and rectify them.